azure-maven-plugin-lib/src/test/java/com/microsoft/azure/maven/artifacthandler/ArtifactHandlerBaseTest.java

```diff
@@ -31,7 +31,7 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class ArtifactHandlerBaseTest {
-    private final AbstractAppServiceMojo mojo = spy(AbstractAppServiceMojo.class);
+    private final AbstractAppServiceMojo mojo = mock(AbstractAppServiceMojo.class);
```

Comment:
In ```mock``` creates a complete mock or fake object while in spy, there is the real object and you just spying or stubbing specific methods of it.

We don't need to call any real method of the AbstractAppServiceMojo in this unit test, use mock to make test safer.